curriculum/challenges/english/13-relational-databases/build-a-celestial-bodies-database-project/build-a-celestial-bodies-database.md

```diff
@@ -4,6 +4,7 @@ title: Build a Celestial Bodies Database
 challengeType: 13
 helpCategory: Backend Development
 url: https://github.com/freeCodeCamp/learn-celestial-bodies-database
+coderoadTutorial: https%3A%2F%2Fraw.githubusercontent.com%2FfreeCodeCamp%2Flearn-celestial-bodies-database%2Fmain%2Ftutorial.json
```

Comment:
I can go either way on this, but why not generate this from `url`?

```js
const {pathname} = new URL(url);
const coderoadTutorial = encodeURIComponent(new URL(pathname, "https://raw.githubusercontent.com"));
```

Reply:
I really don't like the repetition here. Until there's _not_ a 1 -> 1 map from `url` to `coderoadTutorial` let's just use `url`.

Reply:
Yea, I really don't like it either - I was just trying to get it working as fast as possible. I'm not sure I understand exactly when you mean @ojeytonwilliams. You're saying to generate the coderoadTutorial from the url?

Reply:
Yep, what Shaun said, basically.

Reply:
✔️

Reply:
should be resolved.